libraries/commerce/product/reducers/productsById.js

```diff
@@ -108,7 +109,11 @@ export default function productsById(state = {}, action) {
 
       return nextState;
     }
-
+    case RECEIVE_FAVORITES:
```

Comment:
Please check if you can use the `receiveProducts` action for that.

Reply:
done